src/assets/categories.json

```diff
@@ -0,0 +1,8 @@
+{
+  "categories": [
+    "Time and help for another",
+    "Time for the environment",
+    "Time for sharing knowledge",
+    "Professional volunteering"
+  ]
```

Comment:
are you intending to keep this all here or is it just temporary?

Reply:
intending to keep probably